Bottom: 316d205752a1a647a36d6f590b6164008d544971
Top:    632fe0e1fc33272be17db31ebbac30ecfc504c3e
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-03 15:02:11 +0800

Refresh of vhost-introduce-new-ioctls-to

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index b983a54..4e656f8 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -1510,11 +1510,11 @@ static long vhost_net_ioctl(struct file *f, unsigned int ioctl,
 		return vhost_net_set_features(n, features);
 	case VHOST_GET_BACKEND_FEATURES:
 		features = VHOST_NET_BACKEND_FEATURES;
-		if (copy_to_user(featurep, &features, sizeof features))
+		if (copy_to_user(featurep, &features, sizeof(features)))
 			return -EFAULT;
 		return 0;
 	case VHOST_SET_BACKEND_FEATURES:
-		if (copy_from_user(&features, featurep, sizeof features))
+		if (copy_from_user(&features, featurep, sizeof(features)))
 			return -EFAULT;
 		if (features & ~VHOST_NET_BACKEND_FEATURES)
 			return -EOPNOTSUPP;
diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 9501d4c..6f6c42d 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1061,7 +1061,7 @@ ssize_t vhost_chr_write_iter(struct vhost_dev *dev,
 	}
 
 	ret = (type == VHOST_IOTLB_MSG) ? sizeof(struct vhost_msg) :
-		                          sizeof(struct vhost_msg_v2);
+	      sizeof(struct vhost_msg_v2);
 done:
 	return ret;
 }
@@ -1156,7 +1156,7 @@ static int vhost_iotlb_miss(struct vhost_virtqueue *vq, u64 iova, int access)
 	struct vhost_iotlb_msg *msg;
 	bool v2 = vhost_backend_has_feature(vq, VHOST_BACKEND_F_IOTLB_MSG_V2);
 
-	node = vhost_new_msg(vq, v2? VHOST_IOTLB_MSG_V2 : VHOST_IOTLB_MSG);
+	node = vhost_new_msg(vq, v2 ? VHOST_IOTLB_MSG_V2 : VHOST_IOTLB_MSG);
 	if (!node)
 		return -ENOMEM;
